.travis.yml

```diff
@@ -31,7 +31,6 @@ matrix:
   allow_failures:
     # temporary until we find out how to prevent (spurious) build failures
     - env: TARGET="docs"
-    - env: TARGET="linux"
```

Comment:
also, this stil has some builds failing because of memory issues: https://travis-ci.org/openframeworks/openFrameworks/jobs/75839351#L1494